src/app/story/directives/player.component.ts

```diff
@@ -37,7 +37,7 @@ export class PlayerComponent implements OnDestroy {
   private audioUrl: string;
   private imageUrl: string;
   private subscriptionUrl: string;
-  private host: string = 'http://play-staging.prx.tech';
+  private host: string = window['ENV']['PLAY_HOST'];
```

Comment:
Okay, so I may have once more misled you with my description in the ticket.  This `PLAY_HOST` should also be added to `core.env.ts`.  Then you `import { Env } from '../core.env'` into this file, and access it at `Env.PLAY_HOST`.

Looks like you should probably also set a DEFAULTS for this to `https://play.prx.org`.

Bleh - and reading that Env code for the first time in awhile, I realize that I don't really like it very much.  But it'll work for now!